ports/stm32/boards/stm32f401xd.ld

```diff
@@ -9,7 +9,8 @@ MEMORY
     FLASH_ISR (rx)  : ORIGIN = 0x08000000, LENGTH = 16K /* sector 0 */
     FLASH_FS (rx)   : ORIGIN = 0x08004000, LENGTH = 112K /* sectors 1,2,3 are 16K, 4 is 64K */
```

Comment:
This will need to be changed to `LENGTH = 64K`, with an appropriate comment.

Reply:
It's already done, for 4 boards with the old comment.